spec/lib/krikri/harvesters/oai_harvester_spec.rb

```diff
@@ -403,25 +417,6 @@
       end
     end
 
```

Comment:
A couple of things: 
  - the older examples were unit tests on the private method `#record_xml`. These consider that method an implementation detail, and instead focus on the output of `#records`.
  - the old version wanted an empty `OAI::Record` to work with; these use the same test objects as the rest of the suite.